### DISCUSSION TITLE
Chip Modifications

### DISCUSSION THREAD

**Discussion by firecerne** [UkrainianBanderasCat/Digital-Logic-Sim2#10] [General]
Is anyone currently working on the chip modifaction system ?

**Comment by UkrainianBanderasCat** [comment]
No, but tomorrow I will add sort chips and longer names

**Reply by firecerne** [reply]
Ok ! I will work on chip modification. Do you have some idea on how I could do it ?

**Reply by UkrainianBanderasCat** [reply]
Can you please explain to me what you mean by Chip Modification? Is it like edit chips?

**Reply by firecerne** [reply]
Yes exactly , I'm sorry if it didn't really make sense.
